accounts/abi/bind/precompile_contract_template.go

```diff
@@ -397,27 +330,32 @@ func {{decapitalise $contract.Type}}Fallback (accessibleState PrecompileAccessib
 
 // create{{.Contract.Type}}Precompile returns a StatefulPrecompiledContract with getters and setters for the precompile.
 {{if .Contract.AllowList}} // Access to the getters/setters is controlled by an allow list for [precompileAddr].{{end}}
-func create{{.Contract.Type}}Precompile(precompileAddr common.Address) StatefulPrecompiledContract {
-	var functions []*statefulPrecompileFunction
+func create{{.Contract.Type}}Precompile(precompileAddr common.Address) (precompile.StatefulPrecompiledContract, error) {
+	var functions []*precompile.StatefulPrecompileFunction
```

Comment:
Quick question, why do we need to pass in the precompile address, why can't we reference `params.HelloWorldPrecompileAddress`. Is it because it hasn't been created yet?

Reply:
we have changed this, but not reflected to template yet.